tests/parametric/test_crashtracking.py

```diff
@@ -0,0 +1,52 @@
+"""
+Test the crashtracking (RC) feature of the APM libraries.
+"""
+
+import pytest
+import json
+import base64
+
+from utils import bug, context, features, irrelevant, missing_feature, rfc, scenarios, flaky
+
+
+@scenarios.parametric
+@features.crashtracking
+class Test_Crashtracking:
+    @missing_feature(context.library == "java", reason="Not implemented")
+    @missing_feature(context.library == "golang", reason="Not implemented")
+    @missing_feature(context.library == "nodejs", reason="Not implemented")
+    @missing_feature(context.library == "ruby", reason="Not implemented")
+    @missing_feature(context.library == "php", reason="Not implemented")
+    @missing_feature(context.library == "cpp", reason="Not implemented")
+    @missing_feature(context.library == "python", reason="Not implemented")
+    def test_report_crash(self, test_agent, test_library):
+        test_library.crash()
+
+        event = test_agent.wait_for_telemetry_event("logs", wait_loops=400)
+        assert self.is_crash_report(event)
+
+    @missing_feature(context.library == "dotnet", reason="Not implemented")
```

Comment:
Why not implemented?

Reply:
Currently .NET uses the wrong environment variable, there is a pending PR to fix that